_includes/image.html

```diff
@@ -0,0 +1,11 @@
+<p></p> <!-- Spacing -->
+<div class="image">
+  {% if include.href %}<a href="{{ include.href }}">{% endif %}
```

Comment:
target should be _blank for the link.

Reply:
what do you mean? why?

Reply:
This way clicking on the link will open it in a new tab.

Reply:
Clicking needs to open a different, higher res version so the link needs to be explicitly given.

Reply:
mm makes sense

Reply:
@Conor12345 yes. `href` is high res version and `src` is version displayed on website

Reply:
```
{% 
include image.html 
href='https://media.alt-f4.blog/ALTF4/13/base_new_map_og.jpg'  <!-- hires/original -->
src='https://media.alt-f4.blog/ALTF4/13/base_new_map.jpg' <!-- optimised -->
alt='Map view of rail network, showing only an eighth of the outposts.'
caption='Map view of rail network, showing just 27 of the 216 outposts.' 
%}
```